src/ai/backend/client/config.py

```diff
@@ -120,6 +120,20 @@ def _clean_tokens(v: str) -> Tuple[str, ...]:
     return tuple(v.split(','))
 
 
+def _clean_address_map(v: str) -> Mapping[str, str]:
```

Comment:
Would there be any way to reuse the param-type implementation to avoid duplication?

Reply:
I have thought about the same. it is clearly similar with the param-type implementation. how about define a function which validates a string and returns a dictionary, in `client.utils` or `client.helpers`?